src/evented-tokenizer.ts

```diff
@@ -266,18 +266,18 @@ export default class EventedTokenizer {
     endTagName() {
       let char = this.consume();
 
-      if (isSpace(char)) {
-        this.transitionTo(TokenizerState.beforeAttributeName);
-        this.tagNameBuffer = '';
+      if (isSpace(char) && isAlpha(this.peek())) {
```

Comment:
Why do we check if the next char is alpha here? I think the goal is to issue an error if there is white space as the first thing after the `</`, right?

If so, maybe something like:

```js
if (isSpace(char)) {
  if (this.tagNameBuffer === '') {
    this.delegate.reportSyntaxError('closing tag must only contain tagname');
  }
}
```

What do you think?

Reply:
I added the check for whitespace after `</` on lines 490-493 of this file:
```js
...
} else {
    this.transitionTo(TokenizerState.endTagName);
    this.delegate.beginEndTag();
    this.delegate.reportSyntaxError('closing tag cannot contain whitespace before tagname');
}
```

The check on line 269 is specifically looking for attributes after the EndTag's tagname. I made the decision to allow whitespace after the tagname because the HTML spec allows for this 
> 4. After the tag name, there may be one or more ASCII whitespace.

So I'm specifically looking for whitespace followed by and ASCII alpha character (the start of an attribute), which is invalid syntax.


I've you'd prefer to completely disallow any whitespace in closing tags, I'd be happy to update this PR to check for that!

Reply:
> I've you'd prefer to completely disallow any whitespace in closing tags, I'd be happy to update this PR to check for that!

Ya, let's do that.